### ISSUE TITLE
 runtime terminal behaves

### ISSUE THREAD

**Issue by yongrenr** [kuleshov-group/caduceus#33]
Hello! I think your work is very detailed! I ran your code! Since I'm not familiar with pytorch-lightning, I'm curious about how the runtime terminal behaves.
![image](https://github.com/kuleshov-group/caduceus/assets/78602328/789da8e0-bcfe-437a-abc8-6b3e73c9fada)

**Comment by yair-schiff** [comment]
This progress bar is create by lightning. The progress is a combination of train and validation epochs. Loss is the training loss in this case. Please let me know if that answers your question

**Comment by yongrenr** [comment]
> This progress bar was created by lightning. Progress is a combination of training and validation periods. In this case, the loss is the loss of training. Please let me know if this answers your question

If I want to add something else to show in it, do you know any way to modify it?

**Comment by yair-schiff** [comment]
I don't know all the full functionality here but this [link](https://lightning.ai/docs/pytorch/stable/common/progress_bar.html) might help.

When you make `self.log` calls in the trainer there is a parameter called `prog_bar`, set it to `True` to add whatever metric / value you want to display in the progress bar, e.g. [here](https://github.com/kuleshov-group/caduceus/blob/17452f8c4066f5d562fdcfb3e4f28534cdc31015/train.py#L362C13-L362C21)
